### ISSUE TITLE
Moving just-install-update-go to AppVeyor

### ISSUE THREAD

**Issue by lvillani** [just-install/dnsconfig#4]
@geek1011 Is it feasible to move jiup-go away from Travis and to AppVeyor? Travis has been acquired ~1 month ago by a PE firm and began firing all senior engineers, which doesn't bode well for users. I would like to move us away from Travis as soon as possible.

**Comment by lvillani** [comment]
OT: I'm also evaluating Azure Pipelines but I would like to consolidate to one CI provider for now, and we use AppVeyor already :)

**Comment by pgaskin** [comment]
Yep, I heard about that too. I had another look at AppVeyor, and since last time I tried it, they seem to have improved greatly and added some features I was looking for which only Travis had before. I'll be moving all my projects on GitHub to AppVeyor, and my projects on my own server to Drone over the next few weeks (but probably most over the next few days).

I'll see if I can do the move for jiup-go sometime today. We would need to request for cron jobs to be enabled.

Also, Azure Pipelines or BuildKite are other options.

**Comment by lvillani** [comment]
BuildKite looks interesting but I totally don't understand whether their offering for open source projects includes the build infrastructure or just the agent and access to their control plane/web UI.

I'm trying Azure Pipelines for a couple of macOS projects. It's interesting but its UI is confusing at times and they don't support caching yet (it should be finally [coming](https://devblogs.microsoft.com/devops/adding-caching-to-azure-pipelines/) in Q1 2019, at last).

**Comment by pgaskin** [comment]
BuildKite only offers the UI and the agent. I can host the agent though.

I agree with you about Azure pipelines.

As for AppVeyor, I think I will probably go with it, at least for now.
